selector-set.js.flow

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+
+declare class SelectorSet<T: any> {
```

Comment:
Declaring `SelectorSet` as a generic has a nice property of ensuring the opaque data you can store along with it is consistent.

``` js
const set = new SelectorSet();
set.add('.foo', 123);
set.matches(el)[0].data // number
```